hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/AssignmentManager.java

```diff
@@ -2280,24 +2295,48 @@ private void addToPendingAssignment(final HashMap<RegionInfo, RegionStateNode> r
    * For system tables, we must assign them to a server with highest version.
    */
   public List<ServerName> getExcludedServersForSystemTable() {
```

Comment:
Is this used?

Reply:
Yes, this is used by HMaster and AM both.